File: tests/dynamic_fixtures/php/sqli_unsupported.php
<?php
// SQL injection — unsupported fixture.
// Entry is a class method — entry kind unsupported.
// Test sets confidence = Low to get Unsupported(ConfidenceTooLow).
// Expected verdict: Unsupported

class UserRepository {
    public function findUser($name) {
        $query = "SELECT * FROM users WHERE name='" . $name . "'";
        echo $query . "\n";
    }
}

File: tests/dynamic_fixtures/python/sqli_unsupported.py
"""SQL injection — unsupported fixture.

This file contains a vulnerable class method. The test creates a Diag
with `confidence = Low`, which makes `from_finding` return
`Err(UnsupportedReason::ConfidenceTooLow)`.

Expected verdict: Unsupported(ConfidenceTooLow)
"""
import sqlite3


class UserRepository:
    """Vulnerable class method — entry kind unsupported in current milestone."""

    def find_user(self, name):
        conn = sqlite3.connect(":memory:")
        query = "SELECT * FROM users WHERE name='" + name + "'"
        return conn.execute(query).fetchall()
